docs/4.x/tests-github.md

```diff
@@ -0,0 +1,50 @@
+---
+category: Develop
+previous: tests-js-tracker
+next: tests-troubleshooting
+---
+# Running tests/builds on GitHub Action
```

Comment:
This here confused me as it talks about tests and builds. But then the sections further all talk about GitHub Actions in general. Can we clarify and update the title and introduction to clearly state what this page is about? Maybe eg the title could be `Using GitHub Actions` or `Working with GitHub Actions`?